src/index.js

```diff
@@ -1,6 +1,32 @@
-/**
-* Put your JavaScript here
-*/
+let state = {
+  todos: [
+    { title: 'First todo', complete: false },
+    { title: 'Second todo', complete: true },
+    { title: 'Third todo', complete: true },
+  ],
+};
+
+document.onload = init();
```

Comment:
This won't do what you think it will. Can you see why?

Also: `document.onload` will work, but it's not the same as the method we used in class (the `DOMContentLoaded` event). What do you suppose the difference is?